File: cdk-from-cfn-testing/expected/bucket/golang/BucketConstruct.go
package main

import (
	cdk "github.com/aws/aws-cdk-go/awscdk/v2"
	s3 "github.com/aws/aws-cdk-go/awscdk/v2/awss3"
	"github.com/aws/constructs-go/constructs/v10"
	"github.com/aws/jsii-runtime-go"
)

type BucketConstructProps struct {
}

type BucketConstruct struct {
	constructs.Construct
}

func NewBucketConstruct(scope constructs.Construct, id string, props *BucketConstructProps) *BucketConstruct {
	construct := constructs.NewConstruct(scope, &id)

	s3.NewCfnBucket(
		construct,
		jsii.String("Bucket"),
		&s3.CfnBucketProps{
		},
	)

	return &BucketConstruct{
		Construct: construct,
	}
}

File: cdk-from-cfn-testing/expected/bucket/python/BucketConstruct.py
from aws_cdk import Stack
import aws_cdk as cdk
import aws_cdk.aws_s3 as s3
from constructs import Construct

class BucketConstruct(Construct):
  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
    super().__init__(scope, construct_id)

    # Resources
    bucket = s3.CfnBucket(self, 'Bucket',
        )


